src/client/options/mod.rs

```diff
@@ -2334,10 +2334,6 @@ mod tests {
         );
     }
 
-    #[cfg_attr(feature = "tokio-runtime", tokio::test)]
-    #[cfg_attr(feature = "async-std-runtime", async_std::test)]
-    async fn with_invalid_read_preference_mode() {}
```

Comment:
I don't know why this empty test was here but it wasn't doing anything so I just deleted it